src/components/search/advancedSearch/advancedSearchSettings/AdvancedSearchSettings.js

```diff
@@ -1,19 +1,22 @@
 import Title from "@/components/base/title";
-import styles from "./AdvancedSearch.module.css";
+import styles from "./AdvancedSearchSettings.module.css";
 import Translate from "@/components/base/translate/Translate";
+import FieldInputContainer from "../fieldInput/TextInputs";
 import { CqlTextArea } from "@/components/search/advancedSearch/cqlTextArea/CqlTextArea";
 
 /**
+ * Contains advanced search fields
  * @returns {React.JSX.Element}
- *
  */
 
-export default function AdvancedSearch() {
+export default function AdvancedSearchSettings() {
```

Comment:
forudser en lille mergekonflikt her :)

Reply:
Jeg kan ændre det tilbage for nu, og så skifte til noget godt navn senere? 
Ville bare ikke have at det hed `/advancedSearch/advancedSearch/advancedSearch.js` 😬

Reply:
Jeg har ændret det tilbage til AdvancedSearch for at undgå merge conflicts :)